includes/payments/functions.php

```diff
@@ -419,177 +419,29 @@ function give_undo_donation( $payment_id ) {
  * @return object $stats Contains the number of payments per payment status.
  */
 function give_count_payments( $args = array() ) {
```

Comment:
@DevinWalker I update this function and removed some unused filter also, let me know what do you think

Reply:
Let me know if you want to add backward compatibility for `give_count_payments_where` and `give_count_payments_join`